samza-core/src/main/java/org/apache/samza/operators/impl/WindowOperatorImpl.java

```diff
@@ -112,11 +112,18 @@ public WindowOperatorImpl(WindowOperatorSpec<M, K, Object> windowOpSpec, Clock c
 
   @Override
   protected void handleInit(Config config, TaskContext context) {
-    WindowInternal<M, K, Object> window = windowOpSpec.getWindow();
 
     KeyValueStore<TimeSeriesKey<K>, Object> store =
         (KeyValueStore<TimeSeriesKey<K>, Object>) context.getStore(windowOpSpec.getOpId());
 
+    if (initializer != null) {
+      initializer.init(config, context);
```

Comment:
Same question as the keyFunction and valueFunction above.

Reply:
My answer would be same as well. We don't see a use case as of today. Adding the support later would only require implementation change, not API change.